packages/webpack-extraction-plugin/src/GriffelCSSExtractionPlugin.test.ts

```diff
@@ -0,0 +1,199 @@
+
+  return new Promise((resolve, reject) => {
+    compiler.run((err, stats) => {
+      if (err) {
+        reject(err);
+        return;
+      }
+
+      if (typeof stats === 'undefined') {
+        reject(new Error('"stats" from Webpack are not available, unknown error...'));
+        return;
+      }
+
+      if (stats.hasErrors()) {
+        reject(stats.toJson().errors![0]);
+        return;
+      }
+
+      const cssFilePath = path.resolve(__dirname, 'griffel.css');
+
+      if (!virtualFsVolume.existsSync(cssFilePath)) {
+        reject(new Error('"griffel.css" does not exist in filesystem'));
+      }
+
+      const cssOutput = virtualFsVolume.readFileSync(cssFilePath, {
+        encoding: 'utf-8',
+      }) as string;
+      const filesList = virtualFsVolume.readdirSync(__dirname) as string[];
+
+      resolve({
+        cssOutput,
+        filesList,
+      });
+    });
+  });
+}
+
+function fixLineEndings(value: string) {
+  return String(value).replace(/\r?\n/g, '\n').trim();
+}
+
+/**
+ * Test utility similar to "babel-plugin-tester".
+ *
+ * See https://webpack.js.org/contribute/writing-a-loader/#testing.
+ */
+function testFixture(fixtureName: string, options: CompileOptions = {}) {
+  it(`"${fixtureName}" fixture`, async () => {
+    const fixturePath = path.resolve(__dirname, '..', '__fixtures__', 'webpack', fixtureName);
+
+    const tsCodePath = path.resolve(fixturePath, 'code.ts');
+    const tsxCodePath = path.resolve(fixturePath, 'code.tsx');
+
+    const inputPath = [fs.existsSync(tsCodePath) && tsCodePath, fs.existsSync(tsxCodePath) && tsxCodePath].find(
+      Boolean,
+    );
+
+    const errorPath = path.resolve(fixturePath, 'error.ts');
+    const expectedError = fs.existsSync(errorPath) && require(errorPath);
+
+    const fsSnapshotPath = path.resolve(fixturePath, 'fs.json');
+    const cssOutputPath = path.resolve(fixturePath, 'output.css');
+
+    if (!inputPath) {
+      throw new Error(`Failed to find "code.{js,ts,tsx}" in "${fixturePath}"`);
+    }
+
+    if (!fs.existsSync(fsSnapshotPath)) {
+      throw new Error(`Failed to find "fs.json" in "${fixturePath}"`);
+    }
+
+    if (!fs.existsSync(cssOutputPath)) {
+      throw new Error(`Failed to find "output.css" in "${fixturePath}"`);
+    }
+
+    if (!cssOutputPath && !expectedError) {
+      throw new Error(`Failed to find "output.css" or "error.ts" in "${fixturePath}"`);
+    }
+
+    if (expectedError) {
+      if (!expectedError.default) {
+        throw new Error(
+          `Please check that "error.ts" contains a default export with an error or regex in "${fixturePath}"`,
+        );
+      }
+    }
+
+    let resultCSS = '';
+    let resultFsSnapshot: string[] = [];
+
+    let resultError: Error | webpack.StatsError = new Error();
+
+    try {
+      const result = await compileSourceWithWebpack(inputPath, options);
+
+      resultCSS = fixLineEndings(
+        prettier.format(result.cssOutput, {
```

Comment:
is `result.cssOuput` set by the mini-css-extract plugin ?

Reply:
Kinda, `mini-css-extract` plugin extracts CSS to `griffel.css`, `result.cssOutput` is a content of that file.